plugins/zstack.py

```diff
@@ -137,14 +146,15 @@ def _update_exp_dur(self, _=None):
         """
         Called when VA that affects the expected duration is changed
         """
-        nsteps = self.numberofAcquisitions.value
+        nsteps = self.numberOfAcquisitions.value
         step_time = self._estimate_step_duration()
         ss = self._get_acq_streams()
+        nb_acqs = 1 if self._streams_intertwined else len(ss)
```

Comment:
what is `nb_acqs`?

Reply:
Yes, it's a bad name, as there is a VA called "numberOfAcquisitions" which is stored in nsteps. I just basically ran out of words to describe the 3 levels of loop.
I've refactored the code to just not use a name ;-)